### ISSUE TITLE
Is it possible for me to retexture/remove this?

### ISSUE THREAD

**Issue by TheButterbrotMan** [ate47/cursormod#13]
![image](https://user-images.githubusercontent.com/64548098/178145200-3d48cc25-9cee-4233-a357-b793dfa61bac.png)

**Comment by kirmerk1** [comment]
Yes. Open the JAR file of the mod. In the GUI folder there is Click0/1/2.png files. You can either delete the frames you don't like or make a recource pack for this mod without these frames.
